docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md

```diff
@@ -50,11 +48,19 @@ Only a subset of account types and deployment targets support being created dyna
 Before you can create dynamic targets in an Environment, the environment needs to be configured to allow it. See [Enabling dynamic infrastructure](/docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md#enable-dynamic-infrastructure) for more information.
 :::
 
+:::hint
+**Targets from Step Packages**
+Octopus Deploy has recently developed a new architecture for deployment steps and targets, known as step packages. Step packages are available in Octopus Deploy version 2021.3 and later. To learn more about step packages, read the [step package documentation](https://github.com/octopusdeploy/step-api#overview).
```

Comment:
@andrewabest how would a user identify a step package vs other steps?

Reply:
At the moment there isn't a good way to do so aside from reading the docs - in the `new-octopustarget` document I mention that not all targets are from step packages, and the ones that are are in that doc. I'm going to think about this more along with our "first class API experience" work as I feel it all ties together a bit.

Reply:
it makes me wonder if this "hint" even need to be here?
it feels a bit out of context, this is about creating dynamic targets

Reply:
You're probably right, it is a little out of context. The same hint is available in the other document, so I'll remove it from this one.

Reply:
Done